keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -1041,8 +1217,12 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
+None.
+
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+Roll back the feature by disabling the ReadWriteOncePod feature gate.
```

Comment:
This would halt any scheduling of pods using the enhancement. Should we also recommend CSI driver and/or sidecar downgrades?

Reply:
I'm thinking of suggesting to delete any stuck pods and corresponding PVCs. We can also recommend CSI driver and sidecar downgrades but only once the stuck pods are deleted.

Reply:
Would it halt? I think from the comment above, it looks like scheduler will not enforce if the feature gate is disabled.

Reply:
Tested this out, the scheduler will proceed to schedule the pods after the feature is disabled. I'm thinking the order of operations is:

- Delete any unhealthy pods / PVCs using ReadWriteOncePod
- Disable the feature gate (target the API server first to prevent creation of new PVCs)
- Downgrade CSI sidecars and drivers if you're seeing elevated errors there